docs/content/docs/libs/state_processor_api.md

```diff
@@ -515,6 +515,28 @@ Before you interrogate state using the table API, make sure to review our [Flink
 
 IMPORTANT NOTE: State Table API only supports keyed state.
 
+### Metadata
+
+The SQL connector allows users to read metadata of savepoints in the following way:
+```SQL
+LOAD MODULE state;
+SELECT * FROM savepoint_metadata('/root/dir/of/checkpoint-data/chk-1');
+```
+
+The table have the following fixed schema:
```

Comment:
nit:
The table have the following fixed schema: 
->
the new table function creates a table with the following fixed schema

Reply:
Changed.